### PR TITLE
Noto Serif Toto: Version 2.001 added

### PR DESCRIPTION
**Pull request by simoncozens**
b95480c: [gftools-packager] Noto Serif Toto: Version 2.001 added

* Noto Serif Toto Version 2.001 taken from the upstream repo https://github.com/notofonts/toto.git at commit https://github.com/notofonts/toto/commit/.

### CODE REVIEW

**Comment by simoncozens** [comment]
This has no article or description. We need one from @twardoch!

File: ofl/notoseriftoto/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+axes {
+  tag: "wght"
+  min_value: 400.0
+  max_value: 700.0
+}
+source {
+  repository_url: "https://github.com/notofonts/toto.git"
+  archive_url: "https://github.com/notofonts/toto/releases/download/NotoSerifToto-v2.001/NotoSerifToto-v2.001.zip"
+}
+is_noto: true
```
Comment (RosaWagner) [inline ofl/notoseriftoto/METADATA.pb:27]: @simoncozens Missing subsets and languages?

Comment (simoncozens) [reply ofl/notoseriftoto/METADATA.pb:27]: We don't have a language definition or sample text for it. :-/

Comment (RosaWagner) [reply ofl/notoseriftoto/METADATA.pb:27]: Ah, then we can't really merge it, without a specific subset it will only serve Latin, which is a bit useless :/

Comment (simoncozens) [reply ofl/notoseriftoto/METADATA.pb:27]: I'll work on creating one in gflanguages.

**Comment by RosaWagner** [comment]
Noto for self:

Waiting for resolutions on:
- [ ] https://github.com/googlefonts/glyphsets/pull/83
- [x] #5489

**Pull request merged**